docs/src/content/docs/ru/reference/configuration.mdx

```diff
@@ -677,3 +681,117 @@ starlight({
 	credits: true,
 });
 ```
+
+## Настройка коллекций контента
+
+Starlight использует [коллекции контента](https://docs.astro.build/ru/guides/content-collections/) Astro для загрузки вашего контента.
+Загрузчики контента и схемы Starlight помогают настроить коллекции в соответствии с вашими потребностями.
+
+```js
+// src/content.config.ts
+import { defineCollection } from 'astro:content';
+import { docsLoader, i18nLoader } from '@astrojs/starlight/loaders';
+import { docsSchema, i18nSchema } from '@astrojs/starlight/schema';
+
+export const collections = {
+	docs: defineCollection({ loader: docsLoader(), schema: docsSchema() }),
+	// Необязательно: коллекция i18n используется для перевода пользовательского интерфейса на многоязычных сайтах
+	i18n: defineCollection({ loader: i18nLoader(), schema: i18nSchema() }),
```

Comment:
```suggestion
	docs: defineCollection({ loader: docsLoader(), schema: docsSchema() }),
	// Необязательно: коллекция i18n используется для перевода
	// пользовательского интерфейса на многоязычных сайтах
	i18n: defineCollection({ loader: i18nLoader(), schema: i18nSchema() }),
```

As it's quite longer compared to the English version, something we did on other translations was to span the comment on 2 lines to at least avoid too much scrolling on a wide screen. Any thoughts?

Reply:
I think it's a good idea